config/database.yml

```diff
@@ -0,0 +1,15 @@
+default: &default
```

Comment:
not push this file to Github. You can run `git rm --cached config/database.yml ` to remove it out this pull request.